### PR TITLE
 'cf org'  orgs should be alphabetized for easier scanning

### PR DESCRIPTION
**Pull request by diatmpravin**
Fixed this issue by implementing sort.Sort interface for organization models.

### CODE REVIEW

**Comment by diatmpravin** [comment]
@cfdreddbot I have already signed CLA  and sent on mail.

**Comment by jbayer** [comment]
@goehmen @simonleung8 i can confirm that the signed CLA has been submitted.

@diatmpravin thank you for the contribution!

File: cf/commands/organization/orgs.go [hunk 1]
```diff
@@ -51,6 +53,9 @@ func (cmd ListOrgs) Run(c *cli.Context) {
 	table := cmd.ui.Table([]string{T("name")})
 
 	orgs, apiErr := cmd.orgRepo.ListOrgs()
+
+	sort.Sort(models.Organizations(orgs))
```
Comment (maximilien) [inline cf/commands/organization/orgs.go]: You should now add a test that verifies that listing organizations is indeed sorted correctly (even on a list of fake organizations).

You need to test-drive this.

**Comment by goehmen** [comment]
Hey @diatmpravin Thanks for this PR.  An engineer on the CLI team just pointed out that this could actually be a breaking change if anyone depends on current order.  I'd like to think no one would do that but we can't count on that being the case.  

After giving this some thought, i'd like to see this reimplemented as a pair of optional flags.  

```
`cf orgs --asc`   --> sorts in (A-Z, 1-9) order
`cf orgs --desc` --> sorts in (Z-A, 9-1) order.  

If i do this, i'd want to do it for all commands that list items:

apps
stacks
services
service-keys
orgs
spaces domains
routes
buildpacks
org-users
space-users
quotas
space-quotas
service-auth-tokens
service-brokers
security-groups
staging-security-groups
running-security-groups
plugins
list-plugin-repos
repo-plugins

Do you have any interest in working on this as a series of PRs?  I can understand if you don't.  

Thanks

**Comment by goehmen** [comment]
I'm going to go ahead and close this.  If you want to think about the --asc/--desc implementation, let me know here and we can revisit/reopen.  Thanks for your efforts here.

**Comment by goehmen** [comment]
@diatmpravin - Wanted to let you know that I've been gathering evidence to see if there is more perceived value here.  Reopening while I take in that evidence.  Maybe you can work on the tests that @maximilien was recommending?

**Comment by diatmpravin** [comment]
@goehmen I would like to work on these series of PRs and test case.

**Comment by diatmpravin** [comment]
@goehmen @maximilien  Shall I write test case first or all above items will come in one PR? what you would suggest?

**Comment by goehmen** [comment]
@diatmpravin - That's great that you are interested in contributing this.  I've been getting feedback from the OSS community via cf-dev.  When we have something solidified, I'll let you know here.  Good?

**Comment by diatmpravin** [comment]
@goehmen I can see that thread on OSS community. It's fine.

**Comment by diatmpravin** [comment]
@goehmen I am following community thread of this issue. Looking people are suggesting to go for option (b) -  default order to descending. 

Shall I start working on this issue?

**Comment by goehmen** [comment]
@diatmpravin Yes - it looks like option b is the preference.  Let's take it one step at a time.  If you want to submit a PR for Orgs (including test cases), I will take that.

**Comment by camelpunch** [comment]
Closing this, as we expect separate PRs.

**Comment by dkoper** [comment]
Note that CAPI has a story on adding a query parameter on the CC side. I prefer to wait and take advantage of that.
https://www.pivotaltracker.com/n/projects/966314/stories/92019786

**Pull request closed without merging**